### ISSUE TITLE
Java dependency injection docs reference outdated default setting

### ISSUE THREAD

**Issue by benmccann** [playframework/playframework#10007]
[The docs say to set `routesGenerator := InjectedRoutesGenerator` to enable dependency injection](https://www.playframework.com/documentation/2.8.x/JavaDependencyInjection#Dependency-injecting-controllers
). I have not done this and dependency injection still works. I think the docs are outdated. The 2.5 migration guide says this is the default. The 2.7 migration guide says the static routes generator was removed. I don't think there even is any other value it's possible to change this setting to? (unless some crazy advanced user created their own routes generator)

### Play Version

2.8

### API

Java

**Comment by benmccann** [comment]
Another thing to check, does the `@` symbol in the routes file still have special meaning?

**Comment by mkurz** [comment]
Yeah, the docs are outdated. The `InjectedRoutesGenerator` is the default one and no other exist anymore (the static one got removed). AFAIK the `@` symbol still has the meaning described in the docs, so no change here.
